app/features/settings/application/metric/SettingsMetric.js

```diff
@@ -0,0 +1,30 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import PickerModal from 'app/shared_features/picker/PickerModal';
+import * as Actions from './SettingsMetricActions';
+
+
+const mapStateToProps = (state) => ({
+    modalShowing: state.settings.editingDefaultMetric,
```

Comment:
rename modalShowing to isModalShowing

Do this for everywhere modalShowing is.

Reply:
Done